### ISSUE TITLE
Displaying default constructors

### ISSUE THREAD

**Issue by cathylee12** [shuzijun/plantuml-parser#21]
I would like to display the default constructors in the diagram, as well for the consistency. If you agree, I can try to add the feature.

**Comment by shuzijun** [comment]
Is there an example to illustrate this function? Currently, constructors are supported.

**Comment by cathylee12** [comment]
Yes, so for example, in _ExampleConstructor_ class, if I don't explicitly call **_public ExampleConstructor()_**, **_<<Create>> ExampleConstructor()_** does not show up in the diagram.

**Comment by shuzijun** [comment]
Ok, but please make it optional.
eg: https://github.com/shuzijun/plantuml-parser/blob/422af2d91c620aabaed076b3d512add43ababb28/plantuml-parser-core/src/main/java/com/shuzijun/plantumlparser/core/ParserConfig.java#LL34C21-L34C37
